tutorials/install-psql-tutorial.md

```diff
@@ -0,0 +1,151 @@
+# Tutorial: How to install psql on Mac, Ubuntu, Debian, Windows
+
+### Introduction
+`psql` is the standard command line interface for interacting with a PostgreSQL 
+or TimescaleDB instance. Here we explain how to install `psql` on various platforms.
+
+### Before you start
+Before you start, you should confirm that you don’t already have `psql` installed. 
+In fact, if you’ve ever installed Postgres or TimescaleDB before, you likely already 
+have `psql` installed.
+
+```bash
+psql --version
+```
+
+### Install on MacOS using Homebrew
+First, install the [Brew Package Manager][brew-package-manager]. Homebrew simplifies
+the installation of software on macOS.
+
+Second, update `brew`. From your command line, run the following commands:
+
+```bash
+brew doctor
+brew update
+brew install libpq
+```
+
+Finally, create a symbolic link to `psql` (and other `libpq` tools) into `/usr/local/bin`
+so that you can reach it from any command on the macOS Terminal.
+
+```bash
+brew link --force libpq ail
+```
+
+### Install on Ubuntu 16.04,18.04 and Debian 9,10
+Install on Ubuntu and Debian using the `apt` package manager:
+
+```bash
+sudo apt-get update
+sudo apt-get install postgresql-client
+```
+
+>:TIP: This only installs the `psql` client and not the PostgreSQL database.
+
+### Install on Windows 10
+We recommend using the installer from [PostgreSQL.org][windows-installer].
+
+### Last step: Connect to your PostgreSQL server
+Let’s confirm that `psql` is installed:
+
+```bash
+psql --version
+```
+
+Now, in order to connect to your PostgreSQL server, we’ll need the following 
+connection parameters:
+- Hostname
+- Port
+- Username
+- Password
+- Database name
```

Comment:
Hmm you dont need all of those as psql has defaults for most of these and some dont apply for unix domain socket connections

Reply:
I lifted this from Ajay's blog post. How should I rephrase this section?

Reply:
i guess for an introduction its fine as it is